route_generator/launch/route_generator.launch

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
+<launch>
+        <param name="route_file_path" type="str" value="/home/qswawrq/CARMAWorkspace/src/route_generator/resource/" />
```

Comment:
Make this file path relative. It would probably be better to set this parameter either through a file or as a launch file argument

Reply:
Sure. I put it as a launch file argument.